chrome/extension/create_styleurl/CreateStyleURL.js

```diff
@@ -293,7 +300,7 @@ class CreateStyleURLContainer extends React.Component {
         hidden={this.state.hidden}
         onExport={this.handleExport}
         onShareChanges={this.handleShareChanges}
-        onHide={this.handleHide}
+        onHide={() => this.toggleBar(false)}
```

Comment:
I'm not consistent here myself, so this is hypocritical -- but slightly better from a performance perspective if you `() => this.toggleBar(false)` into it's own method, because then it's cached rather than a function created during the render function being called.

E.g.

`handleHideBar = () => this.toggleBar(false)`

Then, you'd make that:
```javascript
onHide={this.handleHideBar}
```